pkgs/misc/drivers/dell-2155cdn/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, rpmextract, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "dell-2155cdn-${version}";
+  version = "1.0-1";
+
+  # http://www.dell.com/support/home/ed/en/eddhs1/product-support/product/dell-2155cn-cdn/drivers
```

Comment:
Note, if this is the place where the source can be downloaded, it can also be set into `meta.downloadPage`.